app/controllers/LookupSchemeAdministratorReferenceController.scala

```diff
@@ -56,13 +58,17 @@ class LookupSchemeAdministratorReferenceController @Inject() (
   val lookupResultID  = "psa-lookup-result"
 
   def displaySchemeAdministratorReferenceForm: Action[AnyContent] = actionWithSessionId.async { implicit request =>
-    sessionCacheService
-      .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
-      .flatMap {
-        case Some(PSALookupRequest(psaRef, _)) =>
-          Future.successful(Ok(psa_lookup_scheme_admin_ref_form(psaRefForm.fill(psaRef))))
-        case _ => Future.successful(Ok(psa_lookup_scheme_admin_ref_form(psaRefForm)))
-      }(executionContext)
+    if (appConfig.psalookupjourneyShutterEnabled) {
+      Future.successful(Ok(withdrawnPSALookupJourney()))
+    } else {
+      sessionCacheService
+        .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
+        .flatMap {
+          case Some(PSALookupRequest(psaRef, _)) =>
+            Future.successful(Ok(psa_lookup_scheme_admin_ref_form(psaRefForm.fill(psaRef))))
+          case _ => Future.successful(Ok(psa_lookup_scheme_admin_ref_form(psaRefForm)))
+        }(executionContext)
+    }
   }
 
   def submitSchemeAdministratorReferenceForm: Action[AnyContent] = actionWithSessionId.async { implicit request =>
```

Comment:
Same here: POST endpoints should also take the feature toggle into consideration.